bedrock/firefox/templates/firefox/os/index-2015.html

```diff
@@ -0,0 +1,14 @@
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/os/devices" %}
+
+{% block content %}
+{% set feed_links = firefox_os_feed_links(LANG) %}
```

Comment:
Are we planning on using these links on other pages? If not, why not just add these links in the view?